src/main/protocol-handler/lens-protocol-router-main/open-deep-link-for-url/open-deep-link.injectable.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import lensProtocolRouterMainInjectable from "../lens-protocol-router-main.injectable";
+
+const openDeepLinkInjectable = getInjectable({
```

Comment:
Still don't think this nomincature is correct.

Reply:
Same as above.